src/terminal/clear.lua

```diff
@@ -0,0 +1,84 @@
+local output = require "terminal.output"
+
+local M = {}
+
+-- Creates an ANSI sequence to clear the screen without writing it to the terminal.
```

Comment:
This repo uses `ldoc` to create the documentation from the source code comments.

These comments don't generate any documentation. Can you bring that back?